content/projects/first-personal-website/_index.md

```diff
@@ -59,9 +59,31 @@ Also, we'll be looking out for good developer habits. Please make sure you push
 
 ### Links
 
+ personal_website_instructions
+- How to make a GitHub Page : https://pages.github.com/
```

Comment:
Looks like you added a bunch of links again that are already present, probably when resolving the merge conflicts, please remove the duplicates, make sure you remove the ones that are not formatted correctly for mark down